__docs__/wonder-blocks-labeled-field/labeled-field.stories.tsx

```diff
@@ -310,10 +319,8 @@ const AllFields = (
                 }
             />
 
-            {showSubmitButtonInStory && (
-                <Button onClick={handleSubmit}>Submit</Button>
-            )}
-        </View>
+            {showSubmitButtonInStory && <Button type="submit">Submit</Button>}
```

Comment:
Because of this change, you're able to submit the form using the "Enter" button now too! `?path=/story/packages-labeledfield--validation`